content/release-engineering/codefresh/best-practices.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Codefresh \"Best Practices\""
+description: "Our tips for working with Codefresh"
+tags:
+- best-practices
+---
+
+# Managing Secrets
+
+Use `chamber` to manage secrets. Provision a dedicated IAM user with limited scope by using the  [terraform-aws-iam-chamber-user](https://github.com/cloudposse/terraform-aws-iam-chamber-user) terraform module. Review our guide on [managing secrets with CI/CD]({{< relref "secrets-management/cicd.md" >}}).
+
+# Naming Kubernetes Contexts
+
+Use cluster name. e.g. `us-west-2-staing-cloudposse-co`
```

Comment:
use `example-com`.
We will replace `example.com` to `cloudposse.co` in separate PR

Reply:
why are we not using cluster names in this format: `us-west-2.staging.example.com`?
we did it before for clients

Reply:
When I looked at joany and peerstreet, they were both in the `-` format.